src/components/SearchModal/bridge/CurrencyListBridge.tsx

```diff
@@ -73,7 +73,7 @@ const CurrencyListBridge = memo(function CurrencyListV2({
   )
 
   return (
-    <div style={{ flex: '1', overflow: 'auto', height: '100%' }}>
+    <div style={{ flex: '1', overflow: 'hidden', height: '100%' }}>
```

Comment:
how about scrollY?

Reply:
actually. it's not scrollable div, scrollable is children of this div.

Reply:
tested on firefox, safari, chrome and brave. Trust me :D

Reply:
okay trusted you for this time :v